screencastr.thor

```diff
@@ -71,13 +72,36 @@ class Screencastr < Thor
   end
 
   desc "brand IN_FILE OUT_FILE", "Transcode, watermark, and add bumpers, all in one command"
+  method_option :upload, aliases: "-u", desc: "Upload the file to S3. Will prompt for file path. Default: false", type: :boolean
   def brand(in_file, out_file)
+    if options[:upload]
+      s3_path = ask "Enter a filepath and filename for the S3 upload:"
```

Comment:
I think this could be confusing based on where the path starts (with or without the bucket name in the path). In fact, I'd say we can go further and set the beginning of the path for them (default the path prefix to `bitmakerhq/lessons`).

So perhaps the prompt should be something like: `Enter the course and cohort path for the S3 upload (e.g. 'web-development/2017-12-team-wall-e')`